src/schema/artist/index.js

```diff
@@ -429,20 +427,43 @@ export const ArtistType = new GraphQLObjectType({
             partner_shows: numeral(
               ({ partner_shows_count }) => partner_shows_count
             ),
-            related_artists: numeral(({ id }) =>
-              total(`related/layer/main/artists`, { artist: id })
-            ),
-            articles: numeral(({ _id }) =>
-              positron("articles", {
-                artist_id: _id,
-                published: true,
-                limit: 1,
-                count: true,
-              }).then(({ count }) => count)
-            ),
+            related_artists: numeral(({ related_artists }) => related_artists),
+            articles: numeral(({ articles }) => articles),
           },
         }),
-        resolve: artist => artist,
+        resolve: (
+          {
+            published_artworks_count,
+            forsale_artworks_count,
+            partner_shows_count,
+            follow_count,
+            id,
+            _id,
+          },
+          _options,
+          _request,
+          { rootValue: { articlesLoader, relatedMainArtistsLoader } }
+        ) => {
+          const related_artists = relatedMainArtistsLoader({
+            artist: id,
+            size: 0,
+            total_count: 1,
+          }).then(({ headers }) => parseInt(headers["x-total-count"] || 0, 10))
```

Comment:
This is repeated a few times, this replaces our legacy `total` loaders, which _just_ want to return a count.

In general I find it a weird pattern so decided not to make helpers (it's just a 1-liner), and I feel like that's _easy enough_ for someone to do if they need.

Reply:
Hmm, a helper would make sure that people don’t accidentally request without `size: 0` for instance.

Reply:
Totally, can make a `totalRelatedArtistsLoader` that would do it, but was basically thinking that it's such a weird pattern that you should be asking yourself (going forward), as you implement: do i _really_ need to fetch _just_ a count here? 😄 

And then if so, you would explicitly have to do this....but I see what you mean.

Reply:
Ideally it would use the `relatedArtistLoader` under the hood so that it can decide to batch this together with related artists, if those are fetched as well.

Reply:
In that case, maybe let's keep this as-is? We can probably come back to a refactor of how we do counts/totals if we'd like.

Reply:
Sounds good 👍